features.md

```diff
@@ -163,14 +163,14 @@ Known limitations
 #### Within-file rename
 {:.v7}
 
-The default mode only allows to rename a local symbol (one which is only used in
-current file).
+This mode only allows to rename a local symbol (one which is only used in
+current file). It is enabled with the command-line flag
+`-cross-file-rename=false`.
```

Comment:
so, basically, do `{:.v11}` and new thing and just keep the existing one as is at the bottom?